### ISSUE TITLE
[Bug]: Atuin shadows after install

### ISSUE THREAD

**Issue by yspotts** [atuinsh/atuin#2414]
### What did you expect to happen?

I installed Atuin following this guide: https://docs.atuin.sh/uninstall/, deleting all the directories in the list. I uninstalled via brew (as that was the method of installation). I also removed all references in my `.zshrc` profile and sourced in the profile. I was expected Atuin to be completely uninstalled. 

### What happened?

It seems there are still some shadows of Atuin somewhere and I can't figure out where. 

When I issue any command from the shell, I get this error message:

`_atuin_preexec:2: command not found: atuin`

As noted above, I removed all references to atuin from my zsh profile. 

Is there something I might have missed? Anything else I need to delete?

### Atuin doctor output

```yaml
I have uninstalled Atuin, so there is no output.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ellie** [comment]
Sounds like you still have the shell integration enabled!

Step 3 here should cover you: https://docs.atuin.sh/uninstall/

Mind if I ask why you uninstalled?

**Comment by yspotts** [comment]
@ellie thanks for the response!

Sorry if I'm missing another integration somewhere, but I removed all references to atuin from `.zshrc`. 
Is there another shell integration point I need to look at?

> Mind if I ask why you uninstalled?
Sure! 
* I found I wasnt using the power features of atuin. Just scrolling up when looking for history
* I found that sometimes I got weird results when scrolling up (it would emit weird characters)
* Most immediately, it seems that a bunch of commands were not being captured in history

So in short, it was more trouble that its worth for my use case and the regular history scrolling in zsh is sufficient for my needs.

**Comment by yspotts** [comment]
doh! I feel a little dumb

Looks like I did everything correctly, and sourced in the updated `.zshrc`, however, this doesn't clear the shell environment. When I open a new terminal window, all is good. And for a current window, I can run `exec zsh` which will reload the entire environment. 

All good now - false alarm. Sorry about that!
